class-3/ejercicio-3.js

```diff
@@ -0,0 +1,18 @@
+const higherThanZero = (numA, numB) => {
+    if (numA === 0 || numB === 0) {
+      return "ninguno de los numeros debe ser cero";
+    } else if (numA > 0 && numB > 0) {
+      return 1;
+    } else if (numA < 0 && numB < 0) {
+      return -1;
+    } else if (numA > 0 && numB < 0) {
+      return 0;
+    }
```

Comment:
Creo que esto no haria falta, no?

```suggestion
    }
```